### PR TITLE
Testing sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
- Add css and html to set the footer in place, add a picture to the welcome page, and give padding to the create and edit forms

### CODE REVIEW

**Comment by StephanieFriend** [comment]
- Added a new background color to the welcome index page

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+<body class="welcome-body">
+
  <link rel="stylesheet" href="stylesheets/weclcome.css" title="text/css" />
 <div class="header">
 <h1 id="welcome-header">Welcome to Adopt Dont Shop</h1>
 </div>
-<div class="main-pic">
-  <img src="https://images.pexels.com/photos/406014/pexels-photo-406014.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500">
+ <div class="main-pic">
+<!--<div class="main-pic">-->
```
Comment (tylertomlinson) [inline app/views/welcome/index.html.erb]: Can we delete this before merging?

Comment (StephanieFriend) [reply app/views/welcome/index.html.erb]: Done! :)

**Comment by StephanieFriend** [comment]
- Deleted comments as requested by Tyler
- Fixed missing ending html tags

**Comment by StephanieFriend** [comment]
- Uncommented out flash messages for shelters test
- Removed form tag from the shelter new view page to properly display flash messages

**Pull request merged**